lib/oms.coffee

```diff
@@ -246,12 +249,12 @@ class @['OverlappingMarkerSpiderfier']
     mData = for m in @markers
       {pt: @llToPt(m['_omsData']?.usualPosition ? m.position), willSpiderfy: no}
     for m1, i1 in @markers
-      continue unless m1.getMap()? and m1.getVisible()  # marker not visible: ignore
+      continue unless m1.getMap()? and @isVisibleMarker(m1)  # marker not visible: ignore
       m1Data = mData[i1]
       continue if m1Data.willSpiderfy  # true in the case that we've assessed an earlier marker that was near this one
       for m2, i2 in @markers
         continue if i2 is i1  # markers cannot be near themselves: ignore
-        continue unless m2.getMap()? and m2.getVisible()  # marker not visible: ignore
+        continue unless m2.getMap()? and @isVisibleMarker(m2)  # marker not visible: ignore
```

Comment:
Just a heads up, `AdvancedMarkerElement` does not have `getMap()` anymore and this will result in a `TypeError: m1.getMap() is not a function`

AME does have `map` instead, I've replaced in my copy `m1.getMap()` with `m1.map` and it seems it works fine. (Same for `m2.getMap()` -> `m2.map`)

https://developers.google.com/maps/documentation/javascript/reference/advanced-markers#AdvancedMarkerElement

Reply:
Super - is there action required on my part for this to be merged, inclusive of the getMap() => map change?